### PR TITLE
CCM-2037: Fix padding on errors responses

### PR DESCRIPTION
**Pull request by seanmarjason-nhs**
## Summary

Updated format for error responses:

image

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Checklist
* [x] Brief description of work completed, and any technical decisions made as part of the PR
* [x] PR link added as a comment to the relevant JIRA ticket
* [x] Branch deployed to a dynamic env - link to deployment pipeline
* [x] PR link shared on Slack and/or Teams
* [ ] 1 reviews received

### CODE REVIEW

File: proxies/shared/resources/jsc/SetResponseDefaults.js [hunk 1]
```diff
@@ -17,3 +17,15 @@ context.setVariable("error.header.Cache-Control", "no-cache, no-store, must-reva
 // set our x-content-type-options-header
 context.setVariable("response.header.X-Content-Type-Options", "nosniff");
 context.setVariable("error.header.X-Content-Type-Options", "nosniff");
+
+// format errors object
+try {
+  const content = context.getVariable("error.content")
+  const errors = JSON.parse(content).errors
```
Comment (PhilipSkinner) [inline proxies/shared/resources/jsc/SetResponseDefaults.js:24]: Could we not just `context.setVariable("error.context", JSON.stringify(JSON.parse(content)))`?

Comment (PhilipSkinner) [reply proxies/shared/resources/jsc/SetResponseDefaults.js:24]: Also what about valid content from the message batches endpoint, see `AssignMessage.MessageBatches.Create.Response.xml`?

File: proxies/shared/resources/jsc/SetResponseDefaults.js [hunk 2]
```diff
@@ -17,3 +17,15 @@ context.setVariable("error.header.Cache-Control", "no-cache, no-store, must-reva
 // set our x-content-type-options-header
 context.setVariable("response.header.X-Content-Type-Options", "nosniff");
 context.setVariable("error.header.X-Content-Type-Options", "nosniff");
+
+// format errors and response objects
+try {
+  const errorContent = context.getVariable("error.content")
+  const responseContent = context.getVariable("response.content")
+
+  if (errorContent) context.setVariable("error.content", JSON.stringify(JSON.parse(errorContent)))
```
Comment (PhilipSkinner) [inline proxies/shared/resources/jsc/SetResponseDefaults.js:26]: Does this work without `{` and `}`? I didn't expect rhinojs to be so forgiving for some reason

**Review by PhilipSkinner — APPROVED** [review]

File: proxies/shared/resources/jsc/SetResponseDefaults.js (hunk 2 above)
Comment (seanmarjason-nhs) [reply proxies/shared/resources/jsc/SetResponseDefaults.js:26]: Surprisingly it was ok

**Pull request merged**